migrations/2025-03-04-124442_community-hidden-visibility/down.sql

```diff
@@ -0,0 +1,31 @@
+ALTER TABLE community
+    ADD COLUMN hidden bool NOT NULL;
+
+UPDATE
+    community
+SET
+    (hidden = TRUE,
+        visibility = 'Public')
+WHERE
+    visibility = 'Hidden';
+
+ALTER TYPE community_visibility RENAME TO community_visibility__;
+
+CREATE TYPE community_visibility AS enum (
+    'Public',
+    'LocalOnly',
+    'Private'
+);
+
+ALTER TABLE community
+    ALTER COLUMN visibility DROP DEFAULT;
+
+ALTER TABLE community
+    ALTER COLUMN visibility TYPE community_visibility
+    USING visibility::text::community_visibility;
```

Comment:
This is throwing an error I cant figure out:
```
---- schema_setup::tests::test_schema_setup stdout ----
Error: LemmyError { message: Unknown("Failed to run 2025-03-04-124442_community-hidden-visibility with: operator does not exist: community_visibility = community_visibility__"), inner: Failed to run 2025-03-04-124442_community-hidden-visibility with: operator does not exist: community_visibility = community_visibility__
```

Other migrations use the exact same code, eg: https://github.com/LemmyNet/lemmy/blob/main/migrations/2024-10-29-090055_private-community/down.sql#L14

Reply:
Tried that already, doesnt make a difference.

Reply:
I commented about a few of these weird ones in migrations/2025-02-06-233105_remove_post_sort_type_enums/up.sql . 

Try dropping the default *before* renaming the old enum.

Reply:
Found it, `idx_community_random_number` references the enum and also has to be dropped. Now it complains about wrong column order in the down migration, what a hassle...

Reply:
Done